file_CRUD_operations

RudraMaheshwari opened the pull request:
This is the pull request for the CRUD operations of the file management/handelling assignment provided in KALPVRIKSHA Program.

RudraMaheshwari committed:
file_operations

RudraMaheshwari committed:
Code_updated_According_pr_updates

pritishajain05 commented on file_crud_operations.c:
```diff
@@ -0,0 +1,250 @@
+            case 3: update_user();
+    				break;
+            case 4: delete_user();
+					break;
```
Correct the indentation

RudraMaheshwari replied:
Done

RudraMaheshwari committed:
Updated_code_with_meet_suggestions

1.White Space Error removed
2.Unique ID feature included
3.Redundancy removed
4.Remove and rename functions resolved

RudraMaheshwari committed:
Corrected_indentation
